core/designsystem/src/main/java/com/iteneum/designsystem/theme/Theme.kt

```diff
@@ -5,33 +5,70 @@ import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.darkColorScheme
 import androidx.compose.material3.lightColorScheme
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
 
 private val DarkColorPalette = darkColorScheme(
-    primary = Snow,
-    onPrimary = JuneBud,
-    secondary = Drab,
-    background = EerieBlack,
-    error = RedSalsa,
+    primary = JuneBud,
+    onPrimary = Drab,
+    primaryContainer = Bianca,
+    onPrimaryContainer = EerieBlack,
+    inversePrimary = AtomicTangerine,
+    secondary = MintJulep,
+    onSecondary = EerieBlack,
+    secondaryContainer = SoftPeach,
+    onSecondaryContainer = EerieBlack,
     tertiary = Boulder,
-    surface = MintJulep,
-    onSurface = Bianca,
-    secondaryContainer = AtomicTangerine,
-    onSecondary = DenimBlue,
-    onPrimaryContainer = SoftPeach
+    onTertiary = EerieBlack,
+    tertiaryContainer = SoftPeach,
+    onTertiaryContainer = EerieBlack,
+    background = Bianca,
+    onBackground = EerieBlack,
+    surface = DenimBlue,
+    onSurface = EerieBlack,
+    surfaceVariant = Drab,
+    onSurfaceVariant = EerieBlack,
+    surfaceTint = JuneBud,
+    inverseSurface = EerieBlack,
+    inverseOnSurface = Snow,
+    error = RedSalsa,
+    onError = Snow,
+    errorContainer = RedSalsa,
+    onErrorContainer = Snow,
+    outline = EerieBlack,
+    outlineVariant = SoftPeach,
+    scrim = Color.Transparent
 )
 
 private val LightColorPalette = lightColorScheme(
-    primary = Snow,
-    onPrimary = JuneBud,
-    secondary = Drab,
-    background = EerieBlack,
-    error = RedSalsa,
+    primary = JuneBud,
```

Comment:
The primary color is the color more used in the view, in this case is the MintJulep the lightest color instead the JuneBud